packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -118,10 +121,21 @@ export const FeedContainer = ({
   const isFinder = router.pathname === '/posts/finder';
   const isV1Search =
     searchValue === SearchExperiment.V1 && showSearch && !isFinder;
+  const shouldShowPulse =
+    checkHasCompleted(ActionType.AcceptedSearch) &&
+    !checkHasCompleted(ActionType.UsedSearch);
```

Comment:
Can't this just be one, usedSearch? if not we can always pulse no?

Reply:
If we make it just one, everyone without going through the flow would receive the pulse. Not sure if that was the intended behavior here in the first place.